### DISCUSSION TITLE
[Scam Alert] Yoozbit یوزبیت

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#12] [Announcements]
<div dir="rtl">

**Project Name:** 
Yoozbit

**نام پروژه:** 
یوزبیت

**نماد پروژه:**
YZC

**اتفاقی که افتاده است:**

حدود ۳ سال هست که این سایت و کوین جعلیش به اسم #یوزکوین فعال هست و تقریبا یک‌سال هست که شرکتی به اسم برخط‌سازان اطلاعات یوز هم ثبت کردن. 
برای خرید یوزکوین شدیدا دارن تبلیغات میکنن و وعده‌های پوچ صعود شدید قیمت کوین، سود قطعی، لیست شدن در صرافی‌های معتبر خارجی و فعالیت بین‌المللی میدن.
پلن‌های زیرمجموعه‌گیری و سرمایه‌گذاری هم اجرا میکنن. 
با دستکاری عمدی قیمت و ایجاد سوابق خرید و فروش جعلی برای کوین‌شون یوزکوین، چارت و حجم بازار ایجاد کردن و مدعی هستن از وزارت علوم و وزارت فرهنگ و ارشاد مجوز رسمی فعالیت دارن. 
اخیرا هم عضو نظام صنفی رایانه ای شده اند.

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار به دلیل عدم شفافیت اهداف پروژه در وایت پیپر.
پروژه از توکن های متعدد و گیج کننده ای صحبت می کند که در وایت پیپر توضیح داده نشده است.

**نوع کلاهبرداری:** 
Scam

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**

**نام شرکت(ها):**
برخط‌سازان اطلاعات یوز
**آدرس وب سایت(ها):**

**منابع:**
https://t.me/yoozbit
http://Www.yoozbit.com

**جمع بندی:**

**Comment by mahtamasom** [comment]
یوزبیت توکن سوزی انجام داده است و 99 درصد توکن هاشو سوزانده و از گردش خارج کرده است.
https://yoozbit.com/bits/135653
